src/track/trackmetadatataglib.cpp

```diff
@@ -68,41 +74,54 @@ const QString kFileTypeOggOpus("opus");
 const QString kFileTypeWAV("wav");
 const QString kFileTypeWavPack("wv");
 
+// Preferred picture types for cover art sorted by priority
+const std::array<TagLib::ID3v2::AttachedPictureFrame::Type, 4> kCoverArtID3v2PictureTypes = {
```

Comment:
I actually used the term "kPreferred..." first. Ok, then lets revert it ;)

On 13.07.2016 23:54, Daniel Schürmann wrote:
>
> In src/track/trackmetadatataglib.cpp 
> <https://github.com/mixxxdj/mixxx/pull/981#discussion_r70716034>:
>
> > @@ -68,41 +74,54 @@ const QString kFileTypeOggOpus("opus");
> >  const QString kFileTypeWAV("wav");
> >  const QString kFileTypeWavPack("wv");
> >
> > +// Preferred picture types for cover art sorted by priority
> > +const std::array<TagLib::ID3v2::AttachedPictureFrame::Type, 4> kCoverArtID3v2PictureTypes = {
>
> kCoverArtID3v2PictureTypesByPriority will be more precise, but a bit long.
> How about "kPreferredID3PictureTypes" and "kPreferredVobisPictureTypes"?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/mixxxdj/mixxx/pull/981/files/83633d64c0c9d7095376c786539248a194af434c#r70716034>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/ABZ-aseo9KbORyDEeb38hOsL9KDF_Ke0ks5qVV57gaJpZM4JLylV>.
>